task/buildah-remote/0.2/buildah-remote.yaml

```diff
@@ -552,6 +566,11 @@ spec:
       chmod +x scripts/script-build.sh
 
       if ! [[ $IS_LOCALHOST ]]; then
+        PRIVILEGED_NESTED_FLAG=""
+        if [[ "${PRIVILEGED_NESTED}" == "true" ]]; then
+          ssh $SSH_ARGS "$SSH_HOST"  mkdir -p "$BUILD_DIR/var/tmp"
+          PRIVILEGED_NESTED_FLAG="--privileged --mount type=bind,source=$BUILD_DIR/var/tmp,target=/var/tmp,relabel=shared"
```

Comment:
Why does specifically `/var/tmp` need to be a mount? Could you add a comment for that here?

Reply:
Added a comment for explanation.